src/style.css

```diff
@@ -0,0 +1,121 @@
+* {
+  box-sizing: border-box;
+}
+
+body,
+p,
+h1,
+button {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+}
+
+.card-image {
+  margin: 32px 19px 36px;
+}
```

Comment:
add padding for all sides to the whole card, not separately for the image and for each info item